docs/Getting Started.md

```diff
@@ -0,0 +1,171 @@
+complex engineering problems, Inductiva provides the computational power and simplicity 
+you need. It's perfect for those who want to push the boundaries of simulation-based 
+research without getting bogged down in the technicalities of simulation setup and 
+execution. It turns brain-busting calculations into child's play, so you can spend 
+more time having those 'eureka!' moments and less time waiting for your simulations 
+to run.
+
+### What can you do with Inductiva API?
+With Inductiva, you have the flexibility to scale your simulations according to 
+your needs:
+
+* For a quick demo, there are [ready-to-use simulators]() that allow you to start 
+simulations immediately with your existing configuration files. No fuss over 
+installations or tangled software wires. 
+
+* For those who dream big and simulate bigger, there's the option to [build your own]() 
+digital dream team of machines, known as a ['MachineGroup'](). This feature lets 
+you configure a custom-tailored squad of virtual machines ready to tackle those 
+heavyweight simulations. 
+
+So, whether you're in for a quick demo dash or for comprehensive, detailed analyses, 
+Inductiva is equipped to facilitate your work seamlessly.
+
+### Why choose Inductiva API?
+Because choosing Inductiva means opting for simplicity and efficiency in your 
+simulation work. With just two lines of code, you can activate its powerful features. 
+No need to tangle with technical headaches or a web of dependencies. Inductiva 
+streamlines the process, so can you cut through the clutter and focus on what 
+really matters – *the science*.
+
+## Prerequisites
+
+### Python Installation
+Inductiva requires Python version 3.8 or higher to function properly. If Python 
+is not already installed on your system, follow these steps:
+
+1. Visit the [Python download page](https://www.python.org/downloads/).
+2. Select and download the appropriate Python version for your operating system.
+3. Follow the installation instructions to set up Python on your system.
+
+> **Tip:** Python updates are released annually in October. It's worth noting that 
+it may take a few months for the scientific Python ecosystem to fully adapt to new 
+releases. Therefore, for the best compatibility with Inductiva, consider using a 
+Python version that has been stable for a few months.
+
+### Visualization Tools
+While using the [Inductiva API](), keep in mind that it does not provide built-in 
+visualization tools for the simulators. To visualize your simulation results:
+
+1. Download the results from Inductiva.
+2. Utilize any visualization tool of your choice to analyze and interpret the data. 
+For large-scale, complex 3D visualizations, our team recommends open-source tools 
+like [ParaView](https://www.paraview.org/download/) or [VTK](https://vtk.org/download/).
+
+## Installing Inductiva API
+
+To install the latest Inductiva package release on PyPI with pip, just open your 
+terminal and run:
```

Comment:
should we ask for people to update pip before (@sergiomsantos : I was having that trouble)

Reply:
I think this should go to the troubleshooting section.
Otherwise we should also say that the user should be using an up-to-date version of python, etc.

Reply:
i really agree with this sergio, seems best practice on all repos i've seen so far